bpf/lib/maps.h

```diff
@@ -121,18 +121,22 @@ struct bpf_elf_map __section_maps tunnel_endpoint_map = {
 
 struct bpf_lpm_trie_key6 {
 	struct bpf_lpm_trie_key lpm_key;
+	__u16 pad1;
+	__u16 pad2;
```

Comment:
Why not making this __u32?

Reply:
Originally I was thinking in terms of port + proto, which would need to be split into (at least) two fields anyway. I can fix this up for simplicity.